packages/widgets/src/components/BotsTable/types.ts

```diff
@@ -0,0 +1,33 @@
+import { Interpolation, Theme } from '@emotion/react';
+import { ExchangeType } from './stories/BotsTable/components/Exchange/types';
+import { TradeType } from './stories/BotsTable/components/Trade/types';
+
+export interface PNL {
+  amount: number;
+  currency: string;
+  isIncreasing: boolean;
+}
+
+export interface ROI {
+  amount: number;
+  isIncreasing: boolean;
+}
+
+export interface BotsTableItem {
+  id: number;
+  name: string;
+  creationDate: string;
+  exchange: ExchangeType;
+  status: TradeType;
+  pair: string;
+  pnl: PNL;
+  roi: ROI;
+  isDisabled: boolean;
+}
+
+export interface BotsTableProps {
+  children: React.ReactElement<React.PropsWithChildren<BotsTableItem>>[];
+  columns: string[];
+  className?: string;
+  css?: Interpolation<Theme>;
```

Comment:
Do you need both `className` and `css` here? 
`emotion` converts `css` into `className`.

Reply:
No, we need only `className` prop